### PR TITLE
Added devise and set the redirection to home page after registration

### PR DESCRIPTION
**Pull request by ciupkajakub**
I added the devise gem.
I set the homepage view before creating an account and after. 
I set the redirection to home page after Signing in.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
-
+gem 'devise'
+gem 'rexml', '~> 3.2', '>= 3.2.5'
```
Comment (socjopata) [inline Gemfile:6]: Did you encounter any problems, so you had to set it that strict?

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Comment (socjopata) [inline app/models/user.rb:2]: Please remove the commented out code. As a rule of thumb we do not commit commented out lines, with small exceptions like initializers / config/env files

File: test/fixtures/users.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
```
Comment (socjopata) [inline test/fixtures/users.yml:4]: By default generating a resources comes up with spec/fixture files. IF you don't plan to use it right away for test writing, please remove such files, as they give false impression, that you actually test your code ;-) Furthermore, my plan is to use factory_girl + rspec, so these are obsolete anyway

**Review by socjopata — APPROVED** [review]
Good job. Just a few comments to address and we can merge that

File: Gemfile (hunk 1 above)
Comment (ciupkajakub) [reply Gemfile:6]: I`ve got an "LoadError - cannot load such file -- rexml/document" and as solution i`ve found that line  above.
I didn`t try using only  "gem 'rexml'".

**Pull request merged**